contributors/chairs-and-techleads/process-workflow-guide.md

```diff
@@ -0,0 +1,253 @@
+
+### Making the Most out of Meetings
+
+Most, if not all, SIGs already have templates for organizing bimonthly meetings. [Here is a collection of templates](https://miro.com/templates/team-meetings/) for remote meetings that you might also find useful.
+
+You’ll find some SIG-specific meeting tips in Release Engineering’s WoW agreement. A few more:
+
+- Ask for topics ahead of time to drive focus, async planning, and meeting management. SIGs Architecture, Testing, and API Machinery do this.
+  - Either cancel the meeting if there is no agenda, or use the meeting time to triage.   
+- Split your meeting time into half discussion, half triaging and planning. If you’re struggling to stay on top of your KEPs and workflow, group meetings offer an opportunity to discuss your backlog and delegate work—including KEP follow-up activities—to contributors. SIG Storage tracks KEPs in their regular bimonthly meetings to ensure KEPs don’t linger in Alpha or Beta forever.  
+- Refine and reorganize the meeting agenda during the first few minutes of the meeting. Put small items at the top, longer discussion items at the bottom. This allows people with quick asks and announcements to drop off the call if they’re not invested in the longer discussion items. SIGs Instrumentation and Auth do this regularly.
+- Pay attention to how many people speak during your SIG meetings. If it’s just 2-3 people having a conversation, then consider keeping the discussion short and taking the topic offline. If you think the entire group should participate, structure the conversation to make that possible.
+
+Meetings provide an opportunity to delegate work to contributors and to make group decisions about KEP backlogs, future plans, roadmaps, etc. For this reason, consider keeping discussions about specific topics brief and high-level. The async sidebar option (discussion on a GitHub issue, or a Slack thread) or small-group call is often better for working out details on a specific topic in order to make the most out of group meeting time.
+
+## Triage and Triaging Tools
+
+Triaging is an effective method for improving SIG responsiveness to contributor requests—and, consequently, keeping contributors engaged and motivated. It’s also useful for:
+- keeping on top of incoming work and delegating it to contributors
+- maintaining company agnosticism. “Devs from other SIGs don’t have to ask for any favors in getting PRs to move forward—it’s just part of the process,” API Machinery Chair Fede Bongiovanni says. “Features don't get biased development.”
+
+You can find [comprehensive triaging guidelines](https://github.com/kubernetes/community/blob/c78c26294b8bd137dd5cc0c807aadc728dad6a57/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
```

Comment:
```suggestion
You can find [comprehensive triaging guidelines](/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
```

For links to items in the same repo, we prefer to use the path (also not linking to a specific commit)